### PR TITLE
Update EVM RPC calls for dynamic fees

### PR DESCRIPTION
**Pull request by cgcardona**
Add `eth_baseFee` and `eth_maxPriorityFeePerGas`.

### CODE REVIEW

File: build/avalanchego-apis/contract-chain-c-chain-api.md [hunk 1]
```diff
@@ -110,6 +110,72 @@ curl -X POST --data '{
...
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/bc/C/rpc
+```
+
+#### **Example Response**
+
+```javascript
+{
+    "jsonrpc": "2.0",
+    "id": 1,
+    "result": "0x34630b8a00"
```
Comment (yulin-dong) [inline build/avalanchego-apis/contract-chain-c-chain-api.md:142]: 0x34630b8a00 = 225000000000, should we use a different value for this example? Also should we add that 225000000000 is 225 gwei?

File: build/avalanchego-apis/contract-chain-c-chain-api.md [hunk 2]
```diff
@@ -110,6 +110,72 @@ curl -X POST --data '{
...
+}
+```
+
+### eth_maxPriorityFeePerGas
+
+Get the priority fee needed to be included in a block.
```
Comment (yulin-dong) [inline build/avalanchego-apis/contract-chain-c-chain-api.md:148]: do we have an explanation somewhere to explain the relationship between baseFee and PriorityFee? I think we should have one.

File: build/avalanchego-apis/contract-chain-c-chain-api.md [hunk 3]
```diff
@@ -110,6 +110,72 @@ curl -X POST --data '{
...
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/bc/C/rpc
+```
+
+#### **Example Response**
+
+```javascript
+{
+    "jsonrpc": "2.0",
+    "id": 1,
+    "result": "0x2540be400"
```
Comment (yulin-dong) [inline build/avalanchego-apis/contract-chain-c-chain-api.md:175]: We should explain that 0x2540be400 = 10000000000.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: build/avalanchego-apis/contract-chain-c-chain-api.md (hunk 1 above)
Comment (cgcardona) [reply build/avalanchego-apis/contract-chain-c-chain-api.md:142]: That's the actual fee that I just got from calling the mainnet w/ this RPC call.

**Review by danlaine — APPROVED** [review]

**Comment by danlaine** [comment]
These are already documented as part of Ethereum's API but can't hurt to include it here given all the questions about it

**Pull request merged**